ui/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: dist test
+
+dist:
```

Comment:
Should we also add a small README.md in `ui` directory? Listing what steps to run for running/building ui?

Reply:
Added